blog.md

```diff
@@ -1,26 +1,78 @@
+
+The database backend for ghost is hosted on the percona cluster I have running on the CoreOS nodes and the location and credentials for it are passed into Ghost via environment variables set by the Deis cli when creating the applications.
+
+I wanted to stick to the [GitHub Flow](https://guides.github.com/introduction/flow/) for development as much as possible (the merge/deploy steps are a bit different here, but should be easy enough to follow along with) utilizing GitHub's notification system on Pull Requests and Merges to trigger Jenkins jobs.
+
+I created a `development` branch in the git repository for the application and then used the Deis cli to create two applications [ghost](http://ghost.ci-demo.paulcz.net) from the code in the `master` branch and [stage-ghost](http://stage-ghost.ci-demo.paulcz.net) from the development branch to be my Production and Staging environments.
+
+Both the `development` and `master` branches are protected with GitHub settings that restrict changes from being pushed directly to the branch and any Pull Requests need to pass tests before they can be merged.
+
+## Deis
+
+Deploying applications with Deis is quite easy and very similar to deploying applications to Heroku. As long as your git repo has a `Dockerfile` (or supports being discovered by the [cedar](https://devcenter.heroku.com/articles/cedar) tooling) Deis will figure out what needs to be done to run your application.
 
-## Application
+After creating an application with the Deis cli you simply push to a remote git server managed by Deis to handle uploading code to install/upgrade the application:
 
-The application I chose to demo is the [Ghost blogging platform](https://ghost.org/download/) (literally the blog you are reading right now).   I chose it because it's a fairly simple app with a backing service ( mysql ).  The source including my `Dockerfile` and customizations can be found in the [paulczar/ci-demo](https://github.com/paulczar/ci-demo) github repository.
+```
+$ git clone https://github.com/deis/example-go.git
+$ cd example-go
+$ deis login http://deis.xxxxx.com
+$ deis create helloworld 
+Creating Application... ...
+done, created helloworld
+Git remote deis added
+$ git push deis master
+
+Counting objects: 39, done.
+Delta compression using up to 8 threads.
+Compressing objects: 100% (38/38), done.
+Writing objects: 100% (39/39), 5.17 KiB | 0 bytes/s, done.
+Total 39 (delta 12), reused 0 (delta 0)
+
+-----> Building Docker image
+remote: Sending build context to Docker daemon 5.632 kB
+<<<<<<<   SNIP   >>>>>>>
+-----> Launching... 
+       done, helloworld:v2 deployed to Deis
+       http://helloworld.ci-demo.paulcz.net
+```
+
+Interacting with applications deployed to Deis from Jenkins is thus relatively simple, you just need to make sure you have your ssh key and remote git url available to jenkins.
```

Comment:
I'm not clear on what you're saying here either. 😕 What does "available to Jenkins" mean, exactly?